engine/schema/src/main/java/com/cloud/upgrade/DatabaseUpgradeChecker.java

```diff
@@ -631,6 +635,27 @@ public void check() {
         } finally {
             lock.releaseRef();
         }
+
+        s_logger.info("Running Flyway migration on Cloudstack database");
+        Properties dbProps = DbProperties.getDbProperties();
+        final String cloudUsername = dbProps.getProperty("db.cloud.username");
+        final String cloudPassword = dbProps.getProperty("db.cloud.password");
+        final String cloudHost = dbProps.getProperty("db.cloud.host");
+        final int cloudPort = Integer.parseInt(dbProps.getProperty("db.cloud.port"));
+        final String dbUrl = "jdbc:mysql://" + cloudHost + ":" + cloudPort + "/cloud";
```

Comment:
@ustcweizhou we also have cloud_usage database, how do we handle changes for that?

Reply:
@rhtyd this is a good question. To be honest, I never considered it.
Actually not only me, all sql files in engine/schema/resources/META-INF/db/ do not consider it...